contrib/generate_kind_yaml.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# utility script to generate yaml config file with dynamic number of master 
+# and worker nodes to be used when creating a cluster. Rename the generated file to
+# kind.yaml or kind-ha.yaml as appropriate for the use case
+
+#PRE-REQUISITE: docker needs to be installed to run the docker image --> mikefarah/yq
+
+# Configurable definitons and defaults. The below can be set as environment variables as well
+MASTER_COUNT=${MASTER_COUNT:-1}
+NODE_COUNT=${NODE_COUNT:-2}
+MASTER_COUNT_MAX=${MASTER_COUNT_MAX:-3}
+NODE_COUNT_MAX=${NODE_COUNT_MAX:-15}
+OUTPUT_FILE=${OUTPUT_FILE:-gen_kind.yaml.GEN}
```

Comment:
Consider changing the file name to gen-kind.yaml. Mentioning two changes hare replacing underscore (_) with hyphen (-) which is generally preferred. Then removing the  .GEN suffix from the file. SInce the "gen" prefix is already used I don't think we need to use it as suffix as well and it will be clear we generated a YAML file.